src/main/java/org/jabref/gui/preferences/ExternalTab.fxml

```diff
@@ -0,0 +1,150 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.Button?>
+<?import javafx.scene.control.CheckBox?>
+<?import javafx.scene.control.ComboBox?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.control.RadioButton?>
+<?import javafx.scene.control.TextField?>
+<?import javafx.scene.control.ToggleGroup?>
+<?import javafx.scene.layout.ColumnConstraints?>
+<?import javafx.scene.layout.GridPane?>
+<?import javafx.scene.layout.HBox?>
+<?import javafx.scene.layout.RowConstraints?>
+<?import javafx.scene.layout.VBox?>
+
+<?import javafx.scene.control.Tooltip?>
+<fx:root prefWidth="650.0" spacing="10.0" type="VBox"
+         xmlns="http://javafx.com/javafx/8.0.212" xmlns:fx="http://javafx.com/fxml/1"
+         fx:controller="org.jabref.gui.preferences.ExternalTabView">
+    <fx:define>
+        <ToggleGroup fx:id="openConsole"/>
+        <ToggleGroup fx:id="openPDF"/>
+        <ToggleGroup fx:id="openFileBrowser"/>
+    </fx:define>
+    <children>
+        <Label styleClass="sectionHeader" text="%External programs"/>
+        <Label text="%Sending of emails"/>
```

Comment:
This should also be a section header, right? `styleClass="sectionHeader"`

Reply:
Yes, thanks. And as you remind me, I think I'm going to facelift everything after conversion of the tabs including resorting the preferences a little bit. :-)